ccmlib/scylla_rest.py

```diff
@@ -0,0 +1,27 @@
+
+import json
+import requests
+
+class ScyllaREST():
+    def __init__(self, ip = '127.0.0.1', port = 10000):
+        self.ip = ip
```

Comment:
@slivne, will it work with the way we run tests in parallel?

Reply:
if we don't change the ip it won't ...
- I think the best would be not to have the default at all - so this will fail if not set.
- the other thing that we should do is not expose this class and make it internal in my view - its better if the calls come through a scylla_node and not through a generic class that is not clear how its connected to a cluster - think about nodetool we do not have have a generic nodetool class - we have nodetool affiliated with a cluster which means run on all nodes in the cluster and nodetool affiliated with a node that means run on this node - the same goes for the rest api.

so in my view having an external file and not having it inside scylla_node is bad ....

Reply:
Having a standalone class simply reflects the fact that one can perform a REST API operation without a scylla_node context (which it true). Same logic applies to the default IP address - it is reasonable to try 127.0.0.1 if nothing else is known.
Of course, when scylla_node embeds ScyllaREST, it will make up its own bed.